borgmatic/commands/borgmatic.py

```diff
@@ -269,6 +269,8 @@ def run_actions(
         'repositories': ','.join([repo['path'] for repo in location['repositories']]),
     }
 
+    hook_context['log_file'] = global_arguments.log_file if global_arguments.log_file else ''
```

Comment:
You could fold this directly into the `hook_context` declaration if you wanted. (Less mutation.) Example:

```python
hook_context = {
   ...
   'repositories': ...
   'log_file': global_arguments.log_file if global_arguments.log_file else ''
}
```

I do not feel strongly about this though.